app/views/pets/show.html.erb

```diff
@@ -1,22 +1,22 @@
 <% content_for :title, "#{@pet.name} - Adopt Don't Shop" %>
 
-  <header>
-  </header>
-  <nav>
-    <ul>
-      <li><%= link_to "All Pets", "/pets" %></li>
-      <li><%= link_to "All Shelters", "/shelters" %></li>
-    </ul>
-  </nav>
+
   <section class="show">
     <h1><%= "Meet #{@pet.name}!" %></h1>
     <%= image_tag @pet.image, alt: "Photo of #{@pet.name}", title: "Photo of #{@pet.name}" %>
     <p>About: <%= @pet.description %></p>
     <p>Age: <%= @pet.age %></p>
     <p>Sex: <%= @pet.sex %></p>
-    <p>Adoption Status: <%= @pet.status %></p>
+    <p>Adoption Status:
+      <% if @pet.status ==true %>
```

Comment:
If we want to move this if/else logic elsewhere that works for me. I wasn't sure where else to put it